gp-entry-blocks/gpeb-entry-edit-timestamp.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Gravity Perks // Entry Blocks // Show Edit Entry Timestamp on Entries Block.
+ * http://gravitywiz.com/
+ *
+ * Instruction Video: https://www.loom.com/share/af4e456263324971b3463ff344c77830
+ *
+ */
+add_action( 'gform_after_update_entry', function ( $form, $entry_id ) {
+	update_entry_meta( $entry_id );
+}, 10, 2 );
+
+add_action( 'gform_post_update_entry', function ( $entry, $original_entry ) {
+	update_entry_meta( rgar( $entry, 'id' ) );
+}, 10, 2 );
+
+function update_entry_meta( $entry_id ) {
```

Comment:
This is going to be in the global namespace so it should be scoped to this snippet.